doc/src/sgml/installation-meson.sgml

```diff
@@ -1104,6 +1104,48 @@ meson configure ... --with-llvm LLVM_CONFIG='/path/to/llvm/bin/llvm-config'
 
      <variablelist>
 
+      <varlistentry>
+       <term><option>--auto_features=<replaceable>enabled/disabled/auto</replaceable></option></term>
+       <listitem>
+        <para>
+         Setting this option allows you to override value of all 'auto' features.
+         This can be useful when you want to disable or enable all the "optional"
+         features at once without having to set each of them manually. The default
+         value for this parameter is auto.
+        </para>
+       </listitem>
+      </varlistentry>
+
+      <varlistentry>
+       <term><option>--backend=<replaceable>BACKEND</replaceable></option></term>
+       <listitem>
+        <para>
+         The default backend meson uses is ninja and that should suffice for most use cases.
+         However, if you'd like to use a different build system to build Postgres, you can
+         set it using this option. The accepted values for BACKEND are:
+         <command>ninja,vs,vs2010,vs2012,vs2013,vs2015,vs2017,vs2019,vs2022 and xcode</command>.
```

Comment:
I don't think it's a good idea to list all the vs20* variants here, that'll be guaranteed to be out of date soon. And since xcode isn't known to work... Maybe just add that 'vs' can be used on windows, to fully integrate with visual studio?

Reply:
Meson will anyways detect your VS version and use that, so I'd say it is indeed a good idea to try to elaborate too much here.

Reply:
Can't quite parse your sentence - is there a negation missing?

Reply:
Lol, yeah. :(

Reply:
Changed to "The default backend meson uses is ninja and that should suffice for most use cases. However, if you'd like to fully integrate with visual studio, you can set the BACKEND to vs."